analyzers/nodejs/integration_test.go

```diff
@@ -107,6 +110,16 @@ func initializeProjects(testDir string) error {
 			defer waitGroup.Done()
 
 			projectDir := filepath.Join(testDir, proj.Name)
+			println("initializing " + projectDir)
```

Comment:
Let's get rid of all the `println`s. Use `log.Debug` instead if this is intentional debug logging.